### PR TITLE
Move instruction localization into model

### PR DESCRIPTION
**Pull request by Hamms**
Instructions localization for Blockly levels was formerly being handled
entirely by the levels helper. More subtly, for hardcoded levels, this
meant that knowledge of the actual content of the instructions only
happened at the levels helper level, since for these levels the
instructions live exclusively in the i18n config.

This PR moves the knowledge of the instructions and how to localize them
into the blockly model. In part this is for increased clarity, in part
it is so that features that depend on the instructions - such as TTS -
can reference the instructions text even for hardcoded blockly levels.

### CODE REVIEW

File: dashboard/app/models/blockly.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@
 require 'nokogiri'
 class Blockly < Level
   include SolutionBlocks
+  include LocaleHelper
```
Comment (joshlory) [inline dashboard/app/models/blockly.rb]: A little weird to be including a helper from a model.  Maybe an indication that some of the helper code should also be moved to a model (or Plain Old Ruby Object)?

For `data_t`, you can call `I18n.t` directly.

Comment (Hamms) [inline dashboard/app/models/blockly.rb]: It very likely should be, but given that I'm only including it here for `data_t` I will for this purpose take your suggestion and just call `I18n.t`

**Comment by Hamms** [comment]
PTAL

**Comment by joshlory** [comment]
LGTM!

**Pull request merged**